package.json

```diff
@@ -142,7 +142,6 @@
     "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
     "babel-plugin-transform-object-assign": "^6.22.0",
     "babel-preset-env": "^1.7.0",
-    "common-tags": "^1.8.0",
```

Comment:
I only added this in https://github.com/biglotteryfund/blf-alpha/pull/1123 but in retrospect this was a whole extra dependency for something that could be solved by indenting some code differently.